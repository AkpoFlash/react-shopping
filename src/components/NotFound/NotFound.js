import React from 'react';
import styled from 'styled-components';

import {
	HEADER_HEIGHT,
	FOOTER_HEIGHT,
	GRID_GAP
} from '../../constants/styles';

const ErrorPage = styled.div`
	grid-column: 2;
	grid-row: 2;
	height: calc( 100vh - ${ HEADER_HEIGHT} - ${ FOOTER_HEIGHT } - 2 * ${ GRID_GAP } );
`;

const NotFound = (props) => {
	return (
		<ErrorPage>
			404 - Page not found
		</ErrorPage>
	);
};

export default NotFound;