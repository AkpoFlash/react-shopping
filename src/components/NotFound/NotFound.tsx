import * as React from 'react';
import styled from 'styled-components';

import {
	HEADER_HEIGHT,
	FOOTER_HEIGHT,
	GRID_GAP
} from '../../constants/styles';

const ErrorPage = styled.div`
	grid-column: 2;
	grid-row: 2;
	height: calc( 100vh - ${ HEADER_HEIGHT} - ${ FOOTER_HEIGHT } - 2 * ${ GRID_GAP } );
`;

const NotFound: React.FunctionComponent<any> = (props) => {
	return (
		<ErrorPage>
			404 - Page not found
		</ErrorPage>
	);
};

export default NotFound;