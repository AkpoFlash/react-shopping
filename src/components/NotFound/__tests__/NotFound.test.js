import React from 'react';

import NotFound from '../NotFound';

describe('Dumb NotFound', () => {
	const notFound = shallow(<NotFound />);

	it('check NotFound snapshot', () => {
		expect(notFound).toMatchSnapshot();
	});
});