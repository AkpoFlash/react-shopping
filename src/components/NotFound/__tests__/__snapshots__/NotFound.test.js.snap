// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should NotFound not changed 1`] = `null`;
